app/routes/overview/components/Overview.tsx

```diff
@@ -96,7 +96,7 @@ const Overview = (props: Props) => {
     <Container>
       <Helmet title="Hjem" />
       <Banner
-        header="Abakus har opptak!"
+        header="Abakusrevyen har opptak!"
```

Comment:
```suggestion
        header="Abakusrevyen has opptak!"
```
not sure which is better 🤷‍♀️